cmd/rootMain.go

```diff
@@ -14,6 +15,9 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// rootCommandFlag is associated with the `-c` option of the root command, set in root.go.
+var rootCommandFlag string
```

Comment:
Oh, was thinking in `root.go` b/c my first question was "Where is this defined?" when I saw it there. Guess a comment should go in both places, but I won't block the PR on it.

Reply:
Oh, ok.